README.md

```diff
@@ -1 +1,37 @@
-# next-with-google-secrets
\ No newline at end of file
+# Configure your Node.js Applications with Google Secrets
+
+## Introduction
+
+`@neolution-ch/next-with-google-secrets` starts your next application with predefined mapped google secrets in the config.
```

Comment:
@neotob 

Could you add the options? 

Maybe something like this:
https://github.com/neolution-ch/Neolution.Extensions.Configuration.GoogleSecrets#options

Reply:
yes sure, added 💪 also changed the version from single string to records